data/sea/10-mempool.h

```diff
@@ -1,4 +1,4 @@
-#include "05-error.h"
+#include "06-segv.h"
 
 const size_t iblock_size = 1 * 1024 * 1024;
```

Comment:
gcc doesn't complain about this one though? maybe it just doesn't like ``static const``?